herddb-core/src/test/java/herddb/sql/SimplerPlannerTest.java

```diff
@@ -242,10 +242,13 @@ public void plannerBasicStatementsTest() throws Exception {
                     List<DataAccessor> results = scan.consume();
                     assertEquals(2, results.size());
                     assertEquals(3, results.get(0).getFieldNames().length);
-                    assertEquals(1234L, results.get(0).get(0));
+                    // see https://issues.apache.org/jira/browse/CALCITE-3998
```

Comment:
Interesting improvement by calcite. I read you correspondence on https://issues.apache.org/jira/browse/CALCITE-3998.